zellij-server/src/thread_bus.rs

```diff
@@ -32,14 +30,16 @@ impl ThreadSenders {
                 .unwrap_or_else(|| Ok(()));
             Ok(())
         } else {
-            self.to_screen.as_ref().unwrap().send(instruction)
+            self.to_screen
+                .as_ref()
+                .context("failed to get screen sender")?
+                .send(instruction)
+                .to_anyhow()
```

Comment:
Good catch, initially I implemented it as a crutch for the changes in `tab`, as you noticed correctly. Thank you for removing it in `tab`!

Reply:
I mean, it refused to compile otherwise since `anyhow::Result<T> != Result<T, crossbeam::channel::SendError>` but thanks! :sweat_smile: